requirements.txt

```diff
@@ -1,3 +1,4 @@
-pytest
-selenium
-webdriver_manager
\ No newline at end of file
+pytest~=6.2.4
+selenium~=3.141.0
+webdriver_manager
+requests~=2.25.1
```

Comment:
i agree with these changes, but next time do not include changes that not related to you TC or task.
in other words it should be done with separated PR

Reply:
Ok